.github/workflows/go.yaml

```diff
@@ -16,7 +16,10 @@ jobs:
     - name: Set up Go
       uses: actions/setup-go@v3
       with:
-        go-version: 1.19
+        go-version: 1.21
+
+    - name: Tidy
```

Comment:
Run it manually and include the generated go.sum file. Let's remove this block from here.